acquia_lift.admin.inc

```diff
@@ -0,0 +1,410 @@
+}
+
+function _acquia_lift_admin_build_credential_form(&$form, &$form_state) {
+  $form['acquia_lift_account_id'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Account ID'),
+    '#description' => t('Your Lift subscription\'s account ID.'),
+    '#default_value' => variable_get('acquia_lift_account_id', ''),
+    '#required' => TRUE,
+  );
+  $form['acquia_lift_site_id'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Site ID'),
+    '#description' => t('Current site\'s site ID. WARNING: different sites must use different value here, even between a staging and a production of the same site.'),
+    '#default_value' => variable_get('acquia_lift_site_id', ''),
+    '#required' => TRUE,
+  );
+  $form['acquia_lift_assets_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Assets URL'),
+    '#description' => t('Your Lift application\'s assets URL.  It determines which version of the Lift application is being used.'),
+    '#field_prefix' => 'https://',
+    '#default_value' => _acquia_lift_admin_clean_url(variable_get('acquia_lift_assets_url', '')),
+    '#required' => TRUE,
+  );
+  $form['acquia_lift_decision_api_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Decision API URL'),
+    '#description' => t('Your Lift Decision API\'s URL.  Unless explicitly instructed, leave empty to use default URL.'),
+    '#field_prefix' => 'https://',
+    '#default_value' => _acquia_lift_admin_clean_url(variable_get('acquia_lift_decision_api_url', '')),
+  );
+  $form['acquia_lift_oauth_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Authentication URL'),
+    '#description' => t('Your Lift Authentication API\'s URL.  Unless explicitly instructed, leave empty to use default URL.'),
+    '#field_prefix' => 'https://',
+    '#field_suffix' => '/authorize',
+    '#default_value' => _acquia_lift_admin_clean_url(_acquia_lift_admin_remove_authorize_suffix(variable_get('acquia_lift_oauth_url', ''))),
+  );
+  return $form;
+}
+
+
+/**
+ * Admin form for configuring acquia_lift behavior.
+ */
+function _acquia_lift_admin_build_identity_form(&$form, &$form_state) {
+  $identity_parameter_display_value = variable_get('acquia_lift_identity_param', 'identity');
+  $identity_type_parameter_display_value = variable_get('acquia_lift_identity_type_param', 'identityType');
+  $default_identity_type_display_value = variable_get('acquia_lift_default_identity_type', ACQUIA_LIFT_DEFAULT_IDENTITY_TYPE_IDENTIFIER);
+  $default_identity_type_default_value = variable_get('acquia_lift_default_identity_type', ACQUIA_LIFT_DEFAULT_IDENTITY_TYPE_DEFAULT);
+
+  $form['acquia_lift_identity_param'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Identity Parameter'),
+    '#description' => t('The query string parameter for specific visitor information, such as an email address or social media username, which is sent to the Acquia Lift service. Example: ?<ins>@identity_parameter_display_value</ins>=jdoe01', [
```

Comment:
Yeah - these crossed.  Since it got finalized while this was in code review I can go ahead and update this now.